errormessages/src/ts/main.tsx

```diff
@@ -0,0 +1,110 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+/// <reference path="../tsd/tsd.d.ts" />
+
+import * as cu from 'cu-core';
```

Comment:
First, you're importing this and never using it.
Second, you should only import what you need and not the whole thing from cu-core.  This will matter when our build process supports tree shaking.  ie:  import {client} from 'cu-core';
Third, you should import client which gives you access to the same interface and works the same current as the old cuAPI which you should not be declaring below.  Instead import client and use client.OnAbilityError() to get the same effect. The client class wraps the raw cuAPI and though is very basic right now, it will offer a full wrapper in the future for all cuAPI calls rather than being more or less an alias as it is now.